playbooks/byo/openshift-cluster/enable_dnsmasq.yml

```diff
@@ -0,0 +1,4 @@
+---
+- include: ../../common/openshift-cluster/enable_dnsmasq.yml
+  vars_files:
+  - ../../byo/openshift-cluster/cluster_hosts.yml
```

Comment:
We should add some plays with `include_vars` tasks similar to [byo](https://github.com/openshift/openshift-ansible/blob/506b993d30b46d2106d15a2ed3a46b4de3374818/playbooks/byo/openshift-cluster/config.yml#L2-L16) since v2 doesn't like `vars_files` on includes.